ts/formly/inputs/api-select.html

```diff
@@ -1,5 +1,5 @@
 <ui-select uis-open-close="onDropdownToggle(isOpen)">
-	<ui-select-match placeholder="{{::placeholder}}" allow-clear="true">{{$select.selected.name}}</ui-select-match>
+	<ui-select-match placeholder="{{::placeholder}}" allow-clear="{{allowClear}}">{{$select.selected.name}}</ui-select-match>

```

Comment:
`allow-clear="allowClear"` here

Reply:
the interpolation here is necessary, otherwise the binding is not done :(